server.go

```diff
@@ -274,6 +274,8 @@ type server struct {
 
 	hostAnn *netann.HostAnnouncer
 
+	serverNetParams bitcoinNetParams
```

Comment:
We have `cfg` here which has the full config. This new field is not necessary.

Reply:
See comment about rpcserver above.

Reply:
Agree